templates/xcassets/swift3.stencil

```diff
@@ -3,6 +3,7 @@
 {% if catalogs %}
 {% set colorAlias %}{{param.colorAliasName|default:"AssetColorTypeAlias"}}{% endset %}
 {% set imageAlias %}{{param.imageAliasName|default:"Image"}}{% endset %}
```

Comment:
Note to self: change the default to `ImageTypeAlias` since we're making a breaking release anyway.

Reply:
other note to self: remove this deprecated alias:
https://github.com/orj/SwiftGen/blob/b81d6cd028fc3a7ae5a3d09c853c763a81132a15/Tests/Fixtures/Generated/XCAssets/swift3-context-all-customname.swift#L20

Reply:
And rename this to be similar to the rest:
https://github.com/orj/SwiftGen/blob/b81d6cd028fc3a7ae5a3d09c853c763a81132a15/Tests/Fixtures/Generated/XCAssets/swift3-context-all.swift#L6